pkg/endpoint/endpoint.go

```diff
@@ -188,6 +188,7 @@ const (
 	PolicyGlobalMapName = "cilium_policy"
 )
 
+// LabelsMap holds mapping from numeric policy identity to labels
 type LabelsMap map[policy.NumericIdentity]pkgLabels.LabelArray
```

Comment:
`LabelsMap` can be private, no?

Reply:
It could, but that would mess the nice naming of local variables (labelsMap). Also, after Thomas's KV-store improvements we can cache this centrally (maybe in policy/cache.go), so it would need to be public at that point. In short, I'd like to keep this public for now, too.